bus-mapping/src/evm/opcodes.rs

```diff
@@ -99,7 +100,7 @@ fn fn_gen_associated_ops(opcode_id: &OpcodeId) -> FnGenAssociatedOps {
         // OpcodeId::CALLER => {},
         // OpcodeId::CALLVALUE => {},
         // OpcodeId::CALLDATALOAD => {},
-        // OpcodeId::CALLDATASIZE => {},
+        OpcodeId::CALLDATASIZE => StackOnlyOpcode::<0>::gen_associated_ops,
```

Comment:
I think this can't be just a `StackOnlyOpcode` because it does a `CallContextOp` READ to the `rw_table`, similar to this (from https://github.com/appliedzkp/zkevm-circuits/pull/287): https://github.com/appliedzkp/zkevm-circuits/blob/0e7c1791f849fcb76d6f3367298313e0e5854c4f/bus-mapping/src/evm/opcodes/caller.rs#L21-L26

The `CallContextOp` is pending to be introduced in #292

Reply:
Yes that's true, so let's wait until #292 is merged. Thanks!

Reply:
#292 has been merged :)

Reply:
Thanks, will update my PR tomorrow 👍